### ISSUE TITLE
QM Language

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#25]
Add BP (or maybe rule even) that Quizmasters shouldn't speak explanatory language, pre-question, that could influence strategy.

A QM stating the question number, type, situation (e.g. regular, toss-up, bonus) is parts required and parts allowable/good.
A QM stating "remember you can wait for the whole question", "our 1st QT question!" etc, should not be allowed.

**Comment by scottpeterson** [comment]
I would also include the Rule Book label.

**Comment by kclimenhaga** [comment]
I agree that this is best practice at Internationals; not sure if it needs to be a rule. Do we have quizmasters at Internationals providing this sort of feedback?

**Comment by scottpeterson** [comment]
I do recall this happening rarely at Internationals, though that would've been years ago now. 

In general I don't think something needs to be often violated to require writing it as a rule. Though of course, if it is never violated, adding a rule is likely just adding rule book bloat. I'm indifferent in that scenario.  (this is a pet peeve of mine as QMs often do small things to "be helpful", disregarding how it's patently unfair).

I mostly recall Lana saying things like "That's the quote" (incorrect wording), and "your answer" (On SITs).

**Comment by gryphonshafer** [comment]
I've seen it happen at IBQ, though it was rare and decades ago. (I don't get to watch all that many IBQ quizzes). I've seen it more at the Great West level. Never as an intentionally unfair thing; always as a "let's be helpful and encouraging" thing. But in all cases, it was sloppy QMing. As much as I'm against rule book bloat, I'd love to see this explicitly prohibited because it'll directly result in less sloppy QMing and more fair meets.

**Comment by ARMediting** [comment]
Is there a way to phrase something that's already in the RB more positively/strongly to address this? Like "QMs must only provide the following information"?

**Comment by jswingle** [comment]
I think it fits better as a Best Practice because I'm fine with this being flexible based on quizzing level. I see no issue with district-level quizmasters making lighthearted commentary throughout to make the experience less stressful for new quizzers. At Internationals, it should be more stoic, though without sacrificing a positive demeanor. Even at Internationals I think it should be expected for officials to positively acknowledge quiz-outs or person bonuses, and I don't think it should be considered out of order to occasionally make positive comments about a quiz so long as they are directed toward all teams.

At any rate, it's probably better handled by official selection, with a coach protesting and/or privately talking to the QM after the quiz only if they feel the commentary is excessive. When I first QM'd I received feedback from coaches to this effect, and appreciated it.

**Comment by jswingle** [comment]
@scottpeterson Any interest in progressing this issue further? I'm not super gung-ho on the issue, though I do think it would be a value add to have at least a generically stated best practice, to the effect of "officials should be cautious that any comments made throughout the quiz do not influence strategy" or something like that

**Comment by scottpeterson** [comment]
I would be in favor of something general like that.  Even if it's not "easily challengeable" at least it's something coaches can point to after a quiz

**Comment by jswingle** [comment]
An added bulletpoint at the bottom of "Quizmasters should:" in the Best Practices document, proposal:

> Be conscious not to make any comments that may influence strategy or show favoritism to any one team, even as they maintain a positive, uplifting environment

**Comment by jswingle** [comment]
Really this should be a "should" for ALL officials -- but there isn't presently much of a section in the BP document for the other roles. We can add this to QM for now and expand as the BP document expands and changes.
